### ISSUE TITLE
composer create-project fails

### ISSUE THREAD

**Issue by segler-alex** [treolabs/treopim#491]
I did the following:
```
composer create-project treolabs/skeleton-pim my-treopim-project
```
and it failed with the following messages:
```
Creating a "treolabs/skeleton-pim-demo" project at "./my-treopim-project"
Installing treolabs/skeleton-pim-demo (1.1.2)
  - Installing treolabs/skeleton-pim-demo (1.1.2): Loading from cache
Created project in /var/www/treopim/my-treopim-project
> ComposerCmd::preUpdate
Loading composer repositories with package information

                                                                                                                                         
  [Composer\Downloader\TransportException]                                                                                               
  The "https://packagist.treopim.com/packages.json?id=public" file could not be downloaded: failed to open stream: Connection timed out  
                                                                                                                                         

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--ignore-platform-reqs] [--] [<package>] [<directory>] [<version>]
```

is the your composer repository down?

**Comment by rratsun** [comment]
We fix it in 1.1.3

**Comment by segler-alex** [comment]
thanks this works now
